requirements/constraints.txt

```diff
@@ -11,25 +11,13 @@
 # TODO: Many pinned dependencies should be unpinned and/or moved to this constraints file.
 
 # Use latest Django LTS version
-Django>=2.2,<2.3.0
-
-# django-model-utils version 4.0.0 dropped support to python 2.7	
-django-model-utils<4.0.0	
+Django>=2.2,<3.1
```

Comment:
I think that if we run make upgrade, then the new django version will be 3.0.X is this something expected?

Also, if we change the constraints I think that we should run make upgrade

Reply:
Yes, I was working on Travis changes first. I'll run make upgrade first before getting the reviews on it. 
Thanks for your help.